application/application.go

```diff
@@ -0,0 +1,166 @@
+package application
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/helper"
+	"github.com/josephroberts/edge-node-manager/proxyvisor"
+	"github.com/josephroberts/edge-node-manager/radio"
+)
+
+type ApplicationInterface interface {
+	Process() error
+	loadDevices() (map[string]device.DeviceInterface, error)
+	updateDevices(devices map[string]device.DeviceInterface) error
+	createDevice(localUUID string) (device.DeviceInterface, string, error)
+}
+
+type Application struct {
+	Name   string
+	Device device.SupportedDevice
+	Radio  radio.RadioInterface
+}
+
+func (a *Application) Process() error {
+	log.Printf("-----------------------------------------------------------------\r\n")
+	log.Printf("Application name: %s\r\n", a.Name)
+	log.Printf("Application device type: %s\r\n", a.Device)
+	log.Printf("Application radio type: %T\r\n", a.Radio)
+
+	application, commit, err := helper.GetApplication(config.Persistant, a.Name)
+	if err != nil {
+		log.Printf("No application location or commit found\r\n")
+	} else {
+		log.Printf("Application location: %s\r\n", application)
+		log.Printf("Application commit: %s\r\n", commit)
+	}
+
+	applicationDevices, _ := a.loadDevices()
```

Comment:
You really mustn't ignore errors. If you have a good reason to, comment it. This could result in hard-to-debug issues.